compiler/IREmitter/IREmitter.cc

```diff
@@ -662,6 +662,10 @@ void emitUserBody(CompilerState &base, cfg::CFG &cfg, const IREmitterContext &ir
                     loadYieldParamsResults.insert(bind.bind.variable);
                     /* intentionally omitted, it's part of method preambula */
                 },
+                [&](cfg::YieldParamPresent *i) {
+                    auto *val = Payload::rubyTrue(cs, builder);
+                    Payload::varSet(cs, bind.bind.variable, val, builder, irctx, bb->rubyBlockId);
```

Comment:
This behavior is temporary, and after unifying the behavior for method and block default arguments it will be a no-op.